### DISCUSSION TITLE
Why SDL2 TTF doesn't work on my psp?

### DISCUSSION THREAD

**Discussion by elpoeprod** [pspdev/pspdev#88] [Q&A]
I've tried everything, copied the code from pspdev sample programs, made my own code, even tried other fonts (ttf,otf), but all of that crashes my PSP. 
![image](https://github.com/user-attachments/assets/d6159e39-ef12-4974-8f71-e3fbf7c81bde)
My project includes `SDL2_ttf` and `if(TTF_Init()<0)return 0;` codeline.
IDK what to do, but the font shows fine on PPSSPP emulator. Font file exists in project directory.

**Comment by elpoeprod** [comment]
PSP is a 3008 model, version 6.61, modded with PSP PRO Firmwarre 6.61.

**Comment by sharkwouter** [comment]
Hey @elpoeprod, thanks for reporting this. I think this is probably caused by the following bug: https://github.com/pspdev/psp-packages/issues/45#issuecomment-1286053609

The workaround is to only use the TTF_RenderText_Blended and TTF_RenderUTF8_Blended functions when rendering text. Shaded, Solid and LCD do not work. I did quite a bit of debugging on this issue, but so far I was not able to find another solution unfortunately.
